frontend/sportsmatch-app/src/pages/Index.tsx

```diff
@@ -31,6 +31,9 @@ export default function MainPage() {
   const [page, setPage] = useState<number>(0)
   const size = 5
 
+  console.log(`query`, searchQuery)
```

Comment:
here as well, if you just forgot them during debugging, remove them, if they are there as to serve something useful, keep them